java/com/google/location/lbs/activity/audioset/dataprocessing/CsvParser.java

```diff
@@ -0,0 +1,244 @@
+import java.io.File;
```

Comment:
use package to manage the project.

Reply:
Ok. Thanks! Why don't we include java in front of com.google.location...?